Bug 1802035: openstack UPI: Document Ansible (#3011)
openshift/installer: upi/openstack/README.md

# Ansible Playbooks for Openstack UPI

> This directory contains the Ansible scripts expected to automate most of the command-line work in the [User-Provided-Infrastructure installation](../../docs/user/openstack/install_upi.md).

Reviewer: ```suggestion
This directory contains Ansible that automate the command-line work in the [User-Provided-Infrastructure installation](../../docs/user/openstack/install_upi.md).
```

## How to use

Customize the cluster properties in the [Inventory](./inventory.yaml) file.

**NOTE:** To deploy with Kuryr SDN, update the `os_networking_type` field to `Kuryr`.

> The playbooks in this directory are designed to reproduce an IPI installation (that is, the opinionated installation achieved with the `openshift-install` binary). The reason why they are here is that they might be easier to customise. Depending on the change, it is advised to change the teardown playbooks (`down-*`) accordingly.

Reviewer: ```suggestion
The playbooks in this directory are designed to reproduce an IPI installation, but are highly customizable. Please be aware of changes made to the install playbooks that may require changes to the teardown playbooks.
```
Just to simplify things a little
